app/assets/javascripts/deck_display.js.coffee

```diff
@@ -111,7 +63,30 @@ $(document).on 'turbolinks:load', ->
           $('<i/>', {class: 'fa fa-external-link'})
         )
       else
-        []
+        $('<div/>', {class: 'dropdown float-right dontprint'}).append(
+          $('<a/>', {
+            title: 'Export deck',
+            href: '#',
+            'data-toggle': 'dropdown',
+            'aria-expanded': false
+          }).append(
+            $('<i/>', {class: 'fa fa-upload'})
+          ),
+          $('<div/>', {class: 'dropdown-menu'}).append(
+            $('<a/>', {class: 'dropdown-item', href: '#'})
+              .append('Copy to clipboard in NetrunnerDB format')
+              .on('click', (e) =>
+                e.preventDefault()
+                copyDeckToClipboard(deck))
+          )
+        )
+
+    copyDeckToClipboard = (deck) =>
```

Comment:
this is not actually working for me.  I checked my clipboard site settings and in both incognito.  The code is correct, but something is funny.

More importantly, i wonder if we should support a deep link to the import deck on NRDB with the cards as a param or have a text box expanded to show the cards in that format to copy.

Reply:
Can you think of any way to improve it? Should we go with this until we can improve the integration with NRDB?

Reply:
Can you try the copy to clipboard with the latest version? I'm wonder if I fixed it in https://github.com/Project-NISEI/cobra/pull/179.

Can you think of any way to improve it? Should we go with this until we can improve the integration with NRDB?